app/views/organisation_relationships/managing_agents.html.erb

```diff
@@ -1,23 +1,47 @@
 <% item_label = format_label(@pagy.count, "managing agents") %>
-<% title = "Managing agents" %>
-<% content_for :title, title %>
+
+<% if current_user.data_coordinator? %>
+  <% if params["related_organisation_id"] %>
+    <%= govuk_notification_banner(
+          title_text: "Success",
+          success: true, title_heading_level: 3,
+          title_id: "swanky-notifications"
+        ) do |notification_banner|
+      notification_banner.heading(text: "#{Organisation.find(params["related_organisation_id"]).name} is now one of your managing agents")
+    end %>
+  <% end %>
+<% end %>
+
 <% if current_user.support? %>
   <%= render partial: "organisations/headings", locals: { main: @organisation.name, sub: nil } %>
   <%= render SubNavigationComponent.new(
     items: secondary_items(request.path, @organisation.id),
   ) %>
-  <h2 class="govuk-visually-hidden">managing agents</h2>
+  <h2 class="govuk-visually-hidden">Managing Agents</h2>
+    <% if params["related_organisation_id"] %>
+      <%= govuk_notification_banner(
+            title_text: "Success",
+            success: true, title_heading_level: 3,
+            title_id: "swanky-notifications"
+          ) do |notification_banner|
+        notification_banner.heading(text: "#{Organisation.find(params["related_organisation_id"]).name} is now one of your managing agents")
```

Comment:
small thing, I think this should say "this organisation's" rather than that "your" as the support user might be editing for a different organisation than their own. There's a few other places where the designs say "your" assuming the user is a data coordinator but I think we need some conditional text that shows "this organisation" instead for support users

Reply:
yeah you're totally right! updating